bitnami/metallb/templates/NOTES.txt

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
It would be nice if you can add here information about how to access the application once it is deployed.
Something similar to this one: https://github.com/bitnami/charts/blob/master/bitnami/grafana/templates/NOTES.txt

Reply:
The application is not reachable at all because it is only balancing the traffic.
The only thing which it can be checked with is creating a LoadBalancer ServiceType.